README.md

```diff
@@ -96,7 +97,6 @@ To run the project, just create the database and the schema file should create t
 - [ ] **Processing data in models.**
 - [ ] **Views.**
 - [ ] **Forms.**
-- [ ] **Validations, Model specs.**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you customize your `Getting Started` section by more describing how a new developer could make use of your project. You can this **[template](https://gist.github.com/tresorsawasawa/cc5e9a9e5dd20c0179ae98eca3b1ae96)** as example